src/components/modals/ErrorModal/ErrorModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+ErrorModal.propTypes = {
+  handleErrorOK: PropTypes.func.isRequired,
+}
+
+function ErrorModal({ handleErrorOK }) {
+  return (
+    <ErrorWrapper>
+      <Modal>
+        <ErrorTitle>Ein Fehler ist beim Laden aufgetreten!</ErrorTitle>
+        <InfoMessage>
+          Die App wird zurück gesetzt. Alle Daten sind verloren!
```

Comment:
Maybe change the wording to sound less dramatic. I would be scared to read "Alle Daten sind verloren!"

Reply:
;-) It`s the truth. You can't do anything. This message appears, after the localStorage is corrupt or deleted. So all data is lost anyway ;-) But I will think about it